src/layers/controllers/layersController.ts

```diff
@@ -2,11 +2,12 @@ import { RequestHandler } from 'express';
 import httpStatus from 'http-status-codes';
 import { injectable, inject } from 'tsyringe';
 import { Services } from '../../common/constants';
-import { ILogger, ILayerPostRequest } from '../../common/interfaces';
+import { ILogger, ILayerPostRequest, ILayerToBestRequest } from '../../common/interfaces';
 import { LayersManager } from '../models/layersManager';
 
 type CreateLayerHandler = RequestHandler<undefined, ILayerPostRequest, ILayerPostRequest>;
 type GetLayerHandler = RequestHandler<undefined, ILayerPostRequest>;
+type CreateBestHandler = RequestHandler<undefined, ILayerToBestRequest, ILayerToBestRequest>;
```

Comment:
we said we will remove the term "best" from our code, no?

Reply:
fixed in ee20559